prometheus-operator/manifest.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 name: prometheus-operator
 title: "Prometheus Operator"
-version: 0.32.0
+version: 8.2.0
```

Comment:
This should be the version of the prometheus-operator, not the helm chart.

Reply:
Oops I misunderstood that. Fixed on the next commit.